packages/ai/src/ui-message-stream/handle-ui-message-stream-finish.ts

```diff
@@ -101,20 +121,13 @@ export function handleUIMessageStreamFinish<UI_MESSAGE extends UIMessage>({
       transform(chunk, controller) {
         controller.enqueue(chunk);
       },
+      // @ts-expect-error cancel is still new and missing from types https://developer.mozilla.org/en-US/docs/Web/API/TransformStream#browser_compatibility
```

Comment:
we support node 20 and this was added in 21

<img width="821" height="102" alt="image" src="https://github.com/user-attachments/assets/238b25e7-b86d-4ac1-b001-b55b379d4a0c" />

will this cause any issues on node 20?

Reply:
No issues, it simply uses the previous behaviour where onFinish is not called